src/controller/OperationalCredentialsDelegate.h

```diff
@@ -36,24 +45,27 @@ class DLL_EXPORT OperationalCredentialsDelegate
 
     /**
      * @brief
-     *   This function generates an operational certificate for the given node.
+     *   This function generates an operational certificate for the device.
      *   The API generates the certificate in X.509 DER format.
      *
      *   The delegate is expected to use the certificate authority whose certificate
      *   is returned in `GetIntermediateCACertificate()` or `GetRootCACertificate()`
      *   API calls.
      *
-     * @param[in] peerId       Node ID and Fabric ID of the target device.
-     * @param[in] csr          Certificate Signing Request from the node in DER format.
-     * @param[in] serialNumber Serial number to assign to the new certificate.
-     * @param[in] certBuf      The API will fill in the generated cert in this buffer. The buffer is allocated by the caller.
-     * @param[in] certBufSize  The size of certBuf buffer.
-     * @param[out] outCertLen  The size of the actual certificate that was written in the certBuf.
+     *   The delegate will call `onNOCGenerated` when the NOC is ready.
+     *
+     * @param[in] deviceId        Optional device ID. If provided, the generated NOC must use the provided ID.
```

Comment:
Please document as  Node ID, rather than Device ID

Reply:
Updated the documentation and function signatures to use node ID. Removed serial number from the signature.
Also, added DAC as an input parameter for future use.